impress-org/givewp
# New: Refreshed admin Donors page

Part of #6175 (Scope 2b)

## Description


The donors page needed to be refreshed, and here it is. I accidentally added the filter by form feature, so that comes as a bonus. (That filter still needs to be tweaked a little bit, but should be merged first so we can utilize it on the Donations page.)

## Affects


This PR affects the code for the old donors list, which is in the same file as the single donor view, so please test for any strangeness on the donor view page (click "View Donor" with your cursor over a donor).

## Visuals



https://user-images.githubusercontent.com/7819986/160188786-32f8ce99-ac32-4290-8eb0-a42f45f92377.mp4


## Testing Instructions



1. Install GiveWP
2. In the admin area, click on Donations -> Donors
3. Make sure the Donors list page appears
4. Check that the row actions (View Donors, Delete) work correctly
5. Make sure that searching works correctly

## Pre-review Checklist
